packages/api/src/external/carequality/document/process-outbound-document-retrieval-resps.ts

```diff
@@ -90,7 +107,8 @@ export async function processOutboundDocumentRetrievalResps({
     for (const docRetrievalResp of results) {
       if (docRetrievalResp.documentReference) {
         successDocsRetrievedCount += docRetrievalResp.documentReference.length;
-      } else if (docRetrievalResp.operationOutcome?.issue) {
+      }
+      if (docRetrievalResp.operationOutcome?.issue) {
```

Comment:
shouldnt be an else here since we are returning responses with both documentReferences and operationOutcomes

Reply:
So you are saying that you've seen responses where we have a documentReference and there was an operationOutcome issue?

Reply:
Yes thats how we are constructing them after this PR